cap/jsonschemas/definitions/cms-ntuples-production-v0.0.1.json

```diff
@@ -1,42 +1,54 @@
 {
-    "$schema": "http://json-schema.org/draft-04/schema#",
-    "type": "object",
-    "required": ["code_base", "processing_steps"],
+  "fullname": null, 
+  "experiment": "CMS", 
+  "jsonschema": {
+    "$schema": "http://json-schema.org/draft-04/schema#", 
+    "required": [
+      "code_base", 
+      "processing_steps"
+    ], 
+    "type": "object", 
     "properties": {
-        "code_base": {
-            "title": "Your Code",
-            "type": "string",
-            "x-cap-file": {}
-        },
-        "processing_steps": {
-            "title": "Processing Steps",
-            "type": "array",
-            "required": ["configuration_file", "run_instructions", "output_data"],
-            "items": {
-                "title": "Step",
-                "type": "object",
-                "properties": {
-                    "configuration_file": {
-                        "title": "Configuration File",
-                        "placeholder": "E.g. my_job_config.py",
-                        "type": "string",
-                        "x-cap-file": {}
-                    },
-                    "run_instructions": {
-                        "title": "Run Instructions",
-                        "description": "Provide run instructions",
-                        "placeholder": "E.g. README.md",
-                        "type": "string",
-                        "x-cap-file": {}
-                    },
-                    "output_data": {
-                        "title": "Output",        
-                        "placeholder": "E.g. myfile-data.root",
-                        "type": "string",
-                        "x-cap-file": {}
-                    }
-                }
+      "code_base": {
+        "x-cap-file": {}, 
```

Comment:
We also need to remove this from everywhere, as we are not going to use it anymore.

Reply:
will be handled by #908